source/index.html.erb

```diff
@@ -48,9 +48,9 @@ title: Cornerstone
     <div class="row">
       <div class="col-sm-6">
         <h3>The Cornerstone Experience</h3>
-        <p>Watch this short video to see how and why our approach, which uses realistic job simulations based on
-          research, works better than anything else out there.<br><br>
-        </p>
+        <br>
+        Watch this short video to learn more about our research-based approach for developing future leaders.
+        <br><br>
```

Comment:
the `<p>` tag made the font size different then the rest of the page. The rest of the page goes without a tag and the default font size is greater than a `<p>` tag.